canu/validate/shcd/shcd.py

```diff
@@ -1067,6 +1068,36 @@ def node_list_warnings(node_list, warnings, out="-"):
                 click.secho(f"{', '.join(cell_list)}\n", file=out)
 
 
+def switch_unused_ports(node_list, out):
```

Comment:
This is for CASMNET-974 (shut down unused ports by default), it's currently not used for anything.

Reply:
Is this a thing we want to do? i.e. shut unused ports?

Reply:
@kosonenj We've gone back and forth on this for several releases.  In the past month we've seen 3 systems with "DCHP problems" caused by unused ports not being shut, so this is yet again back on the table.  

I'm up for other solutions because I know this could create problems as well.

Reply:
@lukebates123 after a side-bar conversation with @kosonenj, let's not globally apply this.  Aruba excepting 6300's shuts by default...   I think the ticket that brought this back up was either for Mellanox or Dell solely so this logic should only be applied to those switch types that:
1. have been problematic like the one that leaked dhcp 
2. do NOT have shut by default.

Reply:
I removed from this branch, I added to CASMNET-974 branch.